packages/credentials/src/V1/KiltCredentialV1.ts

```diff
@@ -385,6 +382,28 @@ export async function validateSubject(
       [key.substring(vocab.length)]: value,
     }
   }, {})
+
+  // Turn CType loader & ctypes array into combined loader function
+  const combinedCTypeLoader = newCachingCTypeLoader(
+    cTypes,
+    typeof loadCTypes === 'function'
```

Comment:
I never knew `function` had a typeof. 

Is this new?

Reply:
Mhh I haven't checked, but my assumption is that it's ancient

Reply:
Ok, I checked it was around since JS was around. Never knew.